src/site/stages/build/process-cms-exports/transformers/node-event.js

```diff
@@ -40,7 +40,9 @@ const transform = entity => ({
       }
     : null,
   // The keys of fieldAddress[0] are snake_case, but we want camelCase
-  fieldAddress: mapKeys(entity.fieldAddress[0], (v, k) => camelCase(k)),
+  fieldAddress: entity.fieldAddress[0]
+    ? mapKeys(entity.fieldAddress[0], (v, k) => camelCase(k))
+    : null,
```

Comment:
This change is so that we can require some properties in the `Address` common schema.  If `entity.fieldAddress[0]` is undefined, `mapKeys` will just return an empty object, but in that situation `fieldAddress` should be null.

I don't have _super_ strong feelings about this so I don't mind reverting.